thirdparty/s3.go

```diff
@@ -304,8 +304,7 @@ func NewS3Session(auth *aws.Auth, region aws.Region) *s3.S3 {
 		// create a Transport which includes our TLSConfig with InsecureSkipVerify
 		// and client timeouts.
 		client := util.GetHttpClient()
-		tlsConfig := tls.Config{InsecureSkipVerify: true}
-		client.Transport = &http.Transport{TLSClientConfig: &tlsConfig}
+		client.Transport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
This modifies the object we got from the pool (which is why I assume we're not putting it back in the pool) and we are therefore susceptible to leaks. Should we set InsecureSkipVerify always instead?

Reply:
This is *only* called in/by the API server in response from operations from the S3 Copy Command, so maybe it does, but we don't see any leaks here in the existing case.

Also correct as to why we don't put it back in the pool. 

I'd rather not skip cert validation across the board.